src/precompile.js

```diff
@@ -56,25 +56,7 @@ function precompile(input, opts) {
     else if(pathStats.isDirectory()) {
         var templates = [];
 
-        function addTemplates(dir) {
```

Comment:
http://jslinterrors.com/function-statements-should-not-be-placed-in-blocks

The original code will not break, but it's not good in ES5 because of function scope.